lib/ExpensiMark.js

```diff
@@ -165,9 +165,14 @@ export default class ExpensiMark {
             {
                 // when <pre> and <br /> occur together extra line is added so removing <br />
                 name: 'replacepre',
-                regex: /<\/pre>\s*<br\s*[/]?>/gi,
+                regex: /<\/pre>\s*<br\s*[\/]?>/gi,
                 replacement: '</pre>',
             },
+            {
+                name: 'replacebr',
+                regex: /<br\s*[\/]?><pre>\s*/gi,  
+                replacement: ' <pre>'
+            }
```

Comment:
Add a trailing comma
```suggestion
            },
```

Reply:
Ahh yes, you're right, adding trailing comma now. Also, apologies for these mistakes this is my first PR in Expensify.

Reply:
It's no biggie 😄 Thanks for your quick response!

In future, you could just run `npx eslint foldername --fix` to avoid this.

Reply:
Thanks for your suggestion @rushatgabhane. I would do it next time.